packages/css/src/components/Form/TextField.stories.js

```diff
@@ -25,11 +25,10 @@ const TextField = ({
   size = 'medium',
   state = 'default',
   hint = '',
-  icon = '',
-  iconPosition = 'right',
```

Comment:
Even though this is removed, stories still have icon position  
<img width="1532" alt="Screenshot 2023-02-21 at 11 20 43" src="https://user-images.githubusercontent.com/12156624/220302640-9ddd89c9-133d-4b27-bd85-4220309bbcea.png">